float32 frame_rate
---
bool success
